pkgs/top-level/all-packages.nix

```diff
@@ -23607,10 +23611,10 @@ in
   mynewt-newt = callPackage ../tools/package-management/mynewt-newt { };
 
   inherit (callPackage ../tools/package-management/nix {
-      storeDir = config.nix.storeDir or "/nix/store";
-      stateDir = config.nix.stateDir or "/nix/var";
-      boehmgc = boehmgc.override { enableLargeConfig = true; };
-      })
+    storeDir = config.nix.storeDir or "/nix/store";
+    stateDir = config.nix.stateDir or "/nix/var";
+    boehmgc = boehmgc.override { enableLargeConfig = true; };
+  })
```

Comment:
Can you drop the whitespace-only changes in this file ?

Reply:
Ah, sorry about that, had auto-formatting enabled for nix files, will ensure that is not part of the final squash